python/servo/build_commands.py

```diff
@@ -385,6 +383,6 @@ def copy_file(dll_path: Optional[str]) -> bool:
     windows_sdk_dir = servo.visual_studio.find_windows_sdk_installation_path()
     dll_name = "api-ms-win-crt-runtime-l1-1-0.dll"
     file_to_copy = next(pathlib.Path(windows_sdk_dir).rglob(os.path.join("**", vs_platform, dll_name)))
-    copy_file(file_to_copy)
+    copy_file(str(file_to_copy))
```

Comment:
It might make sense to change the input type rather than cast this to `str`. Not sure. What is the type?